package.json

```diff
@@ -59,11 +59,13 @@
     "@material-ui/core": "5.0.0-alpha.5",
     "@material-ui/icons": "5.0.0-alpha.4",
     "@material-ui/lab": "5.0.0-alpha.5",
+    "@metamask/inpage-provider": "^7.0.0",
     "@msgpack/msgpack": "^1.12.2",
     "@openzeppelin/contracts": "^3.2.0",
     "@servie/events": "^1.0.0",
     "@transak/transak-sdk": "^1.0.25",
     "@types/bn.js": "^4.11.6",
+    "@types/chrome": "^0.0.126",
```

Comment:
Please remove this, `chrome.*` are the same as `browser.* (in function)`

Reply:
ok